docs/notebooks/Training_and_inference_on_an_example_dataset.ipynb

```diff
@@ -67,15 +77,61 @@
+      "--2023-09-01 13:30:33--  https://objects.githubusercontent.com/github-production-release-asset-2e65be/263375180/16df8d00-94f1-11ea-98d1-6c03a2f89e1c?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20230901%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20230901T203033Z&X-Amz-Expires=300&X-Amz-Signature=b9b0638744af3144affdc46668c749128bd6c4f23ca2a1313821c7bbcd54ccdd&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=263375180&response-content-disposition=attachment%3B%20filename%3Ddrosophila-melanogaster-courtship.zip&response-content-type=application%2Foctet-stream\n",
+      "Resolving objects.githubusercontent.com (objects.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...\n",
+      "Connecting to objects.githubusercontent.com (objects.githubusercontent.com)|185.199.108.133|:443... connected.\n",
+      "HTTP request sent, awaiting response... 200 OK\n",
+      "Length: 111973079 (107M) [application/octet-stream]\n",
+      "Saving to: ‘dataset.zip’\n",
+      "\n",
+      "dataset.zip         100%[===================>] 106.79M  63.0MB/s    in 1.7s    \n",
+      "\n",
+      "2023-09-01 13:30:35 (63.0 MB/s) - ‘dataset.zip’ saved [111973079/111973079]\n",
+      "\n",
+      "Archive:  dataset.zip\n",
+      "   creating: dataset/drosophila-melanogaster-courtship/\n",
+      "  inflating: dataset/drosophila-melanogaster-courtship/.DS_Store  \n",
+      "   creating: dataset/__MACOSX/\n",
+      "   creating: dataset/__MACOSX/drosophila-melanogaster-courtship/\n",
+      "  inflating: dataset/__MACOSX/drosophila-melanogaster-courtship/._.DS_Store  \n",
+      "  inflating: dataset/drosophila-melanogaster-courtship/20190128_113421.mp4  \n",
+      "  inflating: dataset/__MACOSX/drosophila-melanogaster-courtship/._20190128_113421.mp4  \n",
+      "  inflating: dataset/drosophila-melanogaster-courtship/courtship_labels.slp  \n",
+      "  inflating: dataset/__MACOSX/drosophila-melanogaster-courtship/._courtship_labels.slp  \n",
+      "  inflating: dataset/drosophila-melanogaster-courtship/example.jpg  \n",
+      "  inflating: dataset/__MACOSX/drosophila-melanogaster-courtship/._example.jpg  \n",
+      "\u001b[01;34mdataset\u001b[00m\n",
+      "├── \u001b[01;34mdrosophila-melanogaster-courtship\u001b[00m\n",
+      "│   ├── \u001b[01;32m20190128_113421.mp4\u001b[00m\n",
+      "│   ├── \u001b[01;32mcourtship_labels.slp\u001b[00m\n",
+      "│   └── \u001b[01;35mexample.jpg\u001b[00m\n",
+      "└── \u001b[01;34m__MACOSX\u001b[00m\n",
+      "    └── \u001b[01;34mdrosophila-melanogaster-courtship\u001b[00m\n",
+      "\n",
+      "3 directories, 3 files\n"
+     ]
+    }
+   ],
    "source": [
     "!apt-get install tree\n",
```

Comment:
The error occurs when running the notebook locally within a conda environment (and then using a pip install command w/in that environment). The notebooks in which I reran training had this error since I needed to use the GPU - I could have rerun the pip install afterwards in a new environment to get rid of the error, but documenting the error as "OK" seemed beneficial.

Reply:
The error occurs when running the notebook locally within a conda environment (and then using a pip install command w/in that environment). The notebooks in which I reran training had this error since I needed to use the GPU - I could have rerun the pip install afterwards in a new environment to get rid of the error, but documenting the error as "OK" seemed beneficial.